### PR TITLE
Recent blocks page

### PR DESCRIPTION
**Pull request by JM066**
Use getStaticProps to load the first page and then the rest using dynamically using getServerSideProps. Also added created some components for the page

### CODE REVIEW

File: pages/blocks/[blockNum]/index.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import BlockDetails from "../../../src/pages/Blocks/BlockDetails";
+import { getBlockDetail } from "../../api/index";
+export default BlockDetails;
+
+export async function getServerSideProps(context: any) {
+  const { params } = context;
+  const blockNum = params.blockNum;
+  const blockDeatail = await getBlockDetail(blockNum);
```
Comment (BruceJi7) [inline pages/blocks/[blockNum]/index.tsx:8]: change to blockDetail plx

File: src/components/Dropdown/index.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  classname,
+  onclick,
+  isDropdownOpen,
+  arrow,
+}: Props) {
+  return (
+    <div className={classNames(styles.Dropdown, classname)}>
+      <div onClick={onclick} className={styles.SelectedItemContainer}>
+        <div>{selected}</div>
+        {!!arrow && arrow}
```
Comment (BruceJi7) [inline src/components/Dropdown/index.tsx:26]: !!arrow should be enough, shouldn't it? hmm

**Review by BruceJi7 — COMMENTED** [review]
One thing to change, a spelling, otherwise, good to go 😄

**Review by BruceJi7 — APPROVED** [review]

File: pages/blocks/[blockNum]/index.tsx (hunk 1 above)
Comment (JM066) [reply pages/blocks/[blockNum]/index.tsx:8]: Oh, gain typos.. sorry.

**Pull request merged**